docs/Developer Resources/Custom Packages.rst

```diff
@@ -210,6 +211,7 @@ packages can be built as follows:
    $ cd zfs
    $ ./configure
    $ make native-deb-utils
+   $ rm ./openzfs-zfs-dracut_*.deb  # deb-based systems usually use initramfs
```

Comment:
You want ../ here, it emits packages in the parent dir, which is why the install command on the next like is `../*.deb`

Reply:
You're right, my sleepy eyes were too closed, that's why I adore reviews :)